spec/rubocop/cop/sorbet/sigils/enforce_sigil_order_spec.rb

```diff
@@ -143,5 +143,50 @@ class Foo; end
           class Foo; end
         RUBY
     end
+
+    it('autocorrects magic comments while removing blank lines') do
+      source = <<~RUBY
+        # frozen_string_literal: true
+
+
+        # typed: true
+
```

Comment:
Let's make sure non empty comment lines are not deleted:

```suggestion
		#
        # typed: true
		#
```